app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -31,7 +31,7 @@
 
               = f.hidden_field :jobs_type, value: @selected_type
 
-      .moj-filter-layout__content class=grid_column_class
+      .moj-filter-layout__content class="grid_column_class"
```

Comment:
Why is this change here?

Reply:
I just happened to come across it whilst I was poking around in the file. The class name should be wrapped in double quotes, no?

Reply:
This is not a class name, it's a method - I presume, otherwise it would be throwing errors.